components/Attendance.tsx

```diff
@@ -37,37 +37,21 @@ const AttendanceTable = ({
               />
             ))
           ) : data?.expireAt ? (
-            <div
-              style={{
-                background: 'var(--background-light)',
-                borderRadius: '22px',
-                height: '280px',
-                display: 'flex',
-                alignItems: 'center',
-                justifyContent: 'center',
-                margin: '12px 24px',
-              }}
-            >
-              <h4
-                style={{
-                  textAlign: 'center',
-                  opacity: '0.8',
-                  fontSize: '16px',
-                }}
-              >
+            <div className="m-[12px_24px] flex h-[280px] items-center justify-center rounded-[22px] bg-backgroundLight">
```

Comment:
Instead of m-[px_px] why not separate them with mx and my ? it makes it way easy to read. still if it works, leave it unattended

Reply:
it works 👍